scpd/lms/templates/body-extra.html

```diff
@@ -0,0 +1,27 @@
+## mako
+
+<%namespace name='static' file='/static_content.html'/>
+
+<div id="su-footer">
+  <div class="su-container">
+    <div class="su-row">
+      <div id="su-footer-logo" class="span2">
+        <a href="http://www.stanford.edu">
+          <img src="https://www.stanford.edu/su-identity/images/footer-stanford-logo@2x.png" alt="Stanford University" width="105" height="49"/>
```

Comment:
This isn't as long as the line below, but maybe we split it up onto multiple lines?

```html
<img
  src="https://www.stanford.edu/su-identity/images/footer-stanford-logo@2x.png"
  alt="Stanford University"
  width="105"
  height="49"
/>
```